apps/hash-frontend/src/components/hooks/use-orgs.ts

```diff
@@ -42,9 +42,9 @@ export const useOrgs = (
       constrainsPropertiesOn: { outgoing: 0 },
       constrainsLinksOn: { outgoing: 0 },
       constrainsLinkDestinationsOn: { outgoing: 0 },
-      isOfType: { outgoing: 1 },
-      hasLeftEntity: { incoming: 1, outgoing: 1 },
-      hasRightEntity: { incoming: 1, outgoing: 1 },
```

Comment:
Just to check if this is the correct way: For a given entity this will traverse the ***incoming*** link. So in the following example, it will also discover `A` when querying for `B`:
```
Entity A --Link-> Entity B
```

Reply:
Yes, that's correct. For orgs we want to retrieve members (users). This is modelled as follows:

```
User -> OrgMembership -> Org
```